### ISSUE TITLE
Is the new payment a bug or is it normal workflow

### ISSUE THREAD

**Issue by valentindragan** [thenbrent/paypal-digital-goods#16]
At the begining, I really thought that  
if( isset( $_GET['PayerID'] ) ) {
            $paypal   = create_example_purchase();
            $response = $paypal->process_payment();
        } else { 
            $paypal   = create_example_subscription();
            $response = $paypal->start_subscription();
        }
is just being used for connection reasons but what it does is create another payment - and the confusion led me into charging my customer another amount than the customer agreed on.
Basically, I can charge my customer anything I want - at least in Sandbox.
Is Paypal really allowing the merchants to charge more then the user agreed on?
Don't get me wrong - i just want to draw my security line in further coding.
